frontend/apps/marketing/src/components/carousels/actionBlockCarousel/ActionBlockCarousel.tsx

```diff
@@ -0,0 +1,133 @@
+'use client';
+
+import {BaseEntry, EntryFields} from 'contentful';
+import '@code-dot-org/component-library/carousel/index.css';
+import React, {ReactNode, useMemo} from 'react';
+
+import ActionBlock, {
+  ActionBlockProps,
+} from '@code-dot-org/component-library/actionBlock';
+import DSCOCarousel from '@code-dot-org/component-library/carousel';
+
+export type ActionBlockCarouselProps = ActionBlockProps & {
+  /** Carousel content w/ fields from Contentful */
+  slides: {
```

Comment:
So in the end it would be awesome to have smth like `slides: SlideContentfulEntry[];`